### PR TITLE
refactor: TTA-49 change date picker

### PR DESCRIPTION
**Pull request by jimmyjaramillo**

### CODE REVIEW

File: src/app/modules/reports/components/time-range-custom/time-range-custom.component.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        [calendarHeaderComponent]="TimeRangeHeader"
+        ></mat-date-range-picker>
+    </mat-form-field>
+    <div class="col-12 col-md-2 my-1">
+        <button type="submit" class="btn btn-primary">Search</button>
+      </div>
+</form>
+
+
+
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-range-custom/time-range-custom.component.html:21]: Leave only one white space at the end of the file

File: src/app/modules/reports/components/time-range-custom/time-range-custom.component.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+  }
+
+  .date-range-form{
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-range-custom/time-range-custom.component.scss:13]: Add a white space at the end of the file

File: src/app/modules/reports/components/time-range-custom/time-range-custom.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
+import { SimpleChange } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MockStore, provideMockStore } from '@ngrx/store/testing';
+import * as moment from 'moment';
+import { ToastrService } from 'ngx-toastr';
+import { EntryState } from 'src/app/modules/time-clock/store/entry.reducer';
+import * as entryActions from '../../../time-clock/store/entry.actions';
+
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-range-custom/time-range-custom.component.spec.ts:9]: Remove white spaces between imports

File: src/app/modules/reports/components/time-range-custom/time-range-custom.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+import { TimeRangeCustomComponent } from './time-range-custom.component';
+
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-range-custom/time-range-custom.component.spec.ts:12]: Add one aditional whitespace after imports

File: src/app/modules/reports/components/time-range-custom/time-range-custom.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  it('When the ngOnChanges method is the first change, the onSubmit method is not called', () => {
+    const userIdNotCalled = 'test-user-2';
+    spyOn(component, 'onSubmit');
+
+    component.ngOnChanges({userId: new SimpleChange(null, userIdNotCalled, true)});
+
+    expect(component.onSubmit).not.toHaveBeenCalled();
+  });
+
+});
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-range-custom/time-range-custom.component.spec.ts:139]: Add a white space at the end of the file

**Review by nicolsss — APPROVED** [review]

**Review by andresacg — APPROVED** [review]
Good job!

**Pull request merged**